README.md

```diff
@@ -1,37 +1,44 @@
+![Lido on Ethereum Logo](./assets/img/lido.png)
+
+<div style="display: flex;" align="center">
+  <img alt="GitHub license" src="https://img.shields.io/github/license/lidofinance/lido-dao?color=orange">
+  <img alt="NodeJS" src="https://img.shields.io/badge/Node_JS-18-blue">
+  <img alt="Solidity" src="https://img.shields.io/badge/solidity-multiver-blue">
+  <img alt="Hardhat" src="https://img.shields.io/badge/hardhat-^2.12-blue">
+  <img alt="Aragon OS" src="https://img.shields.io/badge/Aragon_OS-4.4.0-blue">
+  <img alt="GitHub tests" src="https://img.shields.io/github/actions/workflow/status/lidofinance/lido-dao/linters.yml?label=tests">
+  <img alt="GitHub code analysis" src="https://img.shields.io/github/actions/workflow/status/lidofinance/lido-dao/analyse.yml?label=code analysis">
+  <img alt="GitHub Bytecode" src="https://img.shields.io/github/actions/workflow/status/lidofinance/lido-dao/assert-bytecode.yml?label=assert bytecode">
+</div>
+<br/>
+
+The **Lido on Ethereum Liquid Staking Protocol** allows their users to earn staking rewards on the Beacon chain without locking Ether or maintaining staking infrastructure.
 
 Users can deposit Ether to the Lido smart contract and receive stETH tokens in return. The smart contract then stakes tokens with the DAO-picked node operators. Users' deposited funds are pooled by the DAO, node operators never have direct access to the users' assets.
 
 Unlike staked ether, the stETH token is free from the limitations associated with a lack of liquidity and can be transferred at any time. The stETH token balance corresponds to the amount of Ether that the holder could request to withdraw.
 
-Before getting started with this repo, please read:
-
-* [Documentation](https://docs.lido.fi/)
+**NB:** It's advised to read [Documentation](https://docs.lido.fi/) before getting started with this repo.
 
 ## Lido DAO
 
-The Lido DAO is a Decentralized Autonomous Organization that manages the liquid staking protocol by deciding on key parameters (e.g., setting fees, assigning node operators and oracles, etc.) through the voting power of governance token (DPG) holders.
+The [Lido DAO](https://docs.lido.fi/lido-dao) is a Decentralized Autonomous Organization that manages the liquid staking protocol by deciding on key parameters (e.g., setting fees, assigning node operators and oracles, performing upgrades, etc.) through the voting power of governance token (LDO) holders.
 
-Also, the DAO will accumulate service fees and spend them on insurance, research, development, and protocol upgrades. Initial DAO members will take part in the threshold signature for Ethereum 2.0 by making BLS threshold signatures.
+Also, the Lido DAO accumulates service fees and spends them on research, development, protocol upgrades execution, and hypothetical coverage applications.
```

Comment:
> `accumulates` and `spends`

These two words seem contradictory. Accumulating means to build up and store, and spending implies the opposite.

> hypothetical coverage applications

"hypothetical" is more about imaginary or proposed scenarios used for discussion or analysis, often with no immediate expectation of them becoming real. "Potential," on the other hand, implies a latent quality or capability that could be realized in the future, indicating a possibility that is more grounded in reality than a hypothetical scenario.

> "coverage applications"

A bit ambiguous because the word "application" can have lots of meanings. 

My proposed wording:

The Lido DAO charges service fees that support infrastructure maintenance, research, development, protocol upgrades, and potential loss coverage.